src/common/low_precision_transformations/include/low_precision/propagate_shared_value.hpp

```diff
@@ -117,7 +117,7 @@ class ov::pass::low_precision::PropagateSharedValue : public ov::pass::ModelPass
             if (!dequantization.empty() &&
                 (ov::is_type<opset1::Convert>(dequantization.data.get_node())) &&
                  ov::is_type<opset1::FakeQuantize>(dequantization.data.get_node()->get_input_node_ptr(0))) {
-                inputNode = dequantization.data.get_node()->get_input_node_shared_ptr(0);
```

Comment:
Can fix be applied only where it solve implicit conversion node -> output issues.

Here the pointer obtained by `dequantization.data.get_node()->input_value(0).get_node_shared_ptr();` is same as  `dequantization.data.get_node()->get_input_node_shared_ptr(0);`, if this pointer is used as input for new node the problem will still exists.